lightyear/src/protocol/component.rs

```diff
@@ -1108,10 +1107,10 @@ impl AppComponentExt for App {
         }
     }
 
+    /// Do not use `Time<Fixed>` for `R`. `Time<Fixed>` is already rollbacked.
```

Comment:
This comment probably shouldn't be there, right? This is for users who want to use `add_resource_rollback`

Reply:
Updated the doc comments to add clarity. Let me know what you think!